leonardo-example.conf

```diff
@@ -45,15 +62,6 @@ mysql {
 proxy {
   # Should match the jupyter wildcard cert specified in command above
   jupyterDomain = "JUPYTER_DOMAIN_NAME"
-  # This tag is used to associate the cluster's master with Leonardo's firewall rule.
-  networkTag = "leonardo"
-  # The VPC network that the firewall rule is applied to. Most instances of leo should use
-  # 'default' since it is the VPC allocated in all newly configured GCP projects.
-  firewallVPCNetwork = "default"
```

Comment:
Previously `firewallVPCNetwork` was never used! I made it optional, and set it if it's present (our Leo instance leaves it unset).